edx_proctoring/static/proctoring/js/models/proctored_exam_allowance_model.js

```diff
@@ -7,7 +7,7 @@ edx = edx || {};
     edx.instructor_dashboard.proctoring = edx.instructor_dashboard.proctoring || {};
 
     edx.instructor_dashboard.proctoring.ProctoredExamAllowanceModel = Backbone.Model.extend({
-        url: '/api/edx_proctoring/v1/proctored_exam/allowance'
+        url: '/api/edx_proctoring/v1/proctored_exam/bulk_allowance'
```

Comment:
Instead of making the change directly here, you would need a new Model like this:
`edx.instructor_dashboard.proctoring.ProctoredExamBulkAllowanceModel = Backbone.Model.extend({
        url: '/api/edx_proctoring/v1/proctored_exam/bulk_allowance'`.
Leave the original model intact